### PR TITLE
feat(ecs): support description in ecs instance

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. support description in ecs instance
2. use v1 API to udpate the name and description
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccComputeInstance_basic
=== PAUSE TestAccComputeInstance_basic
=== CONT  TestAccComputeInstance_basic
--- PASS: TestAccComputeInstance_basic (278.19s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       278.300s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_compute_instance.go [hunk 1]
```diff
@@ -78,6 +78,10 @@ func ResourceComputeInstanceV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_compute_instance.go:83]: `Computed` is missing

File: huaweicloud/resource_huaweicloud_compute_instance.go [hunk 2]
```diff
@@ -1030,13 +1036,18 @@ func resourceComputeInstanceV2Update(ctx context.Context, d *schema.ResourceData
 		return diag.Errorf("error creating compute V1.1 client: %s", err)
 	}
 
-	var updateOpts servers.UpdateOpts
+	var updateOpts cloudservers.UpdateOpts
 	if d.HasChange("name") {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_compute_instance.go]: you can use `d.HasChanges("name", "description")` to wrap it

File: huaweicloud/resource_huaweicloud_compute_instance.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_compute_instance.go:83]: if add Computed, we can't update the description to empty

File: huaweicloud/resource_huaweicloud_compute_instance.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_compute_instance.go]: fixed

File: huaweicloud/resource_huaweicloud_compute_instance.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_compute_instance.go:83]: as discussed, Computed added

**Pull request merged**